torchvision/utils.py

```diff
@@ -87,13 +87,16 @@ def norm_range(t, range):
     return grid
 
 
-def save_image(tensor, filename, nrow=8, padding=2,
-               normalize=False, range=None, scale_each=False, pad_value=0):
+def save_image(tensor, fp, nrow=8, padding=2,
```

Comment:
If we add the `format` option, shouldn't we allow all additional parameters (`dpi` for `.png`, ...)for `Image.save()`? We could add a `**kwargs` or `save_kwargs` which is simply passed to `im.save()`.

Reply:
Sure, they could be added based on the use cases. 
IMO file option provides an interface to write as bytes and this might be having a couple of use cases.
What do you think ? @pmeier

Reply:
To be clear: I'm in favour of adding this as you did. I was just asking myself why we should _only_ include  the `format` option although `Image.save()` has a [lot more options](https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.Image.save) for the different file formats. I think if we give the user more control over the options we might as well give him _all_ options. This of course does not need to happen in this PR. I just wanted to put it out there, so I or we don't forget it.

Reply:
@pmeier , I also feel the same, providing a full control over the save would be a better thing for the user.